Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2262,10 +2262,8 @@ def __init__(self, base_url, verify=True, proxy=False, ok_codes=tuple(), headers
             self._headers = headers
             self._auth = auth
             self._session = requests.Session()
-            if proxy:
-                self._proxies = handle_proxy()
-            else:
-                self._proxies = None
+            if not proxy:
```

Comment:
because requests already doing it for us

![image](https://user-images.githubusercontent.com/7270217/71786351-316d9b00-3013-11ea-8ee5-11430788f04c.png)

![image](https://user-images.githubusercontent.com/7270217/71786360-434f3e00-3013-11ea-9237-164af97dbb33.png)